UI/ControlMappingScreen.h

```diff
@@ -78,27 +78,26 @@ class KeyMappingNewKeyDialog : public PopupScreen {
 	bool mapped_;  // Prevent double registrations
 };
 
-class KeyMappingNewMouseKeyDialog : public PopupScreen {
+class BindingChoice;
+
+class KeyMappingDialog : public PopupScreen {
 public:
-	explicit KeyMappingNewMouseKeyDialog(int btn, bool replace, std::function<void(KeyDef)> callback, std::shared_ptr<I18NCategory> i18n)
-		: PopupScreen(i18n->T("Map Mouse"), "", ""), callback_(callback), mapped_(false) {
-		pspBtn_ = btn;
-	}
+	explicit KeyMappingDialog(int btn, std::shared_ptr<I18NCategory> i18n)
+		: PopupScreen(std::string(i18n->T("Mapping for"))+" "+i18n->T(KeyMap::GetPspButtonName(btn))), pspBtn_(btn) {}
```

Comment:
nit: I know this is used elsewhere, but I personally greatly prefer to use for translation:
```c++
std::string label = ReplaceAll(km->T("Currently mapped for %1:"), "%1", mc->T(pspButtonName));
```

Rather than:
```c++
std::string label = km->T("Currently mapped for") + " " + mc->T(pspButtonName) + ":";
```

The reason?  Word order and punctuation can be very tricky to get right the second way.  Another language may prefer to phrase this as "[BUTTON]のマッピング：", which as you can see is in the opposite order and has different punctuation spacing.

-[Unknown]